README.md

```diff
@@ -1,197 +1,212 @@
+The AuthN/Z providers are defined in `package.json` and `config.ts`. You can choose  appropriate providers. SMART allows greater granularity into authentication than Cognito and is the FHIR standard. It allows you to access a FHIR record only if that record has reference to the user. FHIR Works on AWS controls the working of your FHIR server.
 
-## Capabilities
 
-This deployment implementation utilizes Lambda, DynamoDB, S3 and Elasticsearch to provide these FHIR capabilities:
+## FHIR on AWS features
 
-- CRUD operations for all R4 or STU3 base FHIR resources
-- Search capabilities per resource type
-- Ability to do versioned reads (vread)
-- Ability to post a transaction bundle of 25 entries or less
+FHIR Works on AWS solution utilizes AWS Lambda, Amazon DynamoDB, Amazon S3 and Amazon Elasticsearch Service to provide the following FHIR features:
 
-## Quick start/installation
++ Create, Read, Update, Delete (CRUD) operations for all R4 or STU3 base FHIR resources
++ Search capabilities per resource type
++ Ability to do versioned reads ([vread](https://www.hl7.org/fhir/http.html#vread))
++	Ability to post a transaction bundle of 25 entries or less. Presently, transaction bundles with only 25 entries or less are supported.
 
-The easiest and quickest way to experience FHIR Works on AWS is through [AWS solutions](https://aws.amazon.com/solutions/implementations/fhir-works-on-aws/). If you are interested in modifying the code and setting up your developer environment we recommend you following the below instructions:
+## Accessing FHIR Works on AWS
 
-### Download
+The easiest and quickest way to access FHIR Works on AWS is by using [AWS solution](https://aws.amazon.com/solutions/implementations/fhir-works-on-aws/). To modify the code and set up a developer environment, follow the steps below:
 
-Clone or download the repository to a local directory. **Note:** if you intend to modify FHIR Works on AWS you may wish to create your own fork of the GitHub repo and work from that. This allows you to check in any changes you make to your private copy of the solution.
-
-Git Clone example:
+1. Clone or download the repository to a local directory.
+ 
+Example:
 
 ```sh
 git clone https://github.com/awslabs/fhir-works-on-aws-deployment.git
 ```
 
-### Install
+**Note**: To modify FHIR Works on AWS, create your own fork of the GitHub repository. This allows you to check in any changes you make to your private copy of the solution.
```

Comment:
Nitpick: Change `solution` to `code`.

My thought being, we want to use the term `solution` only when referring to the actual [FHIR Solution](https://aws.amazon.com/solutions/implementations/fhir-works-on-aws/)

Reply:
fixed.